Day-11-Quiz-App/quizBuilderCSS.css

```diff
@@ -56,14 +58,58 @@
     color: #fff;
     padding: 10px 0;
     width: 50%;
-    margin: 8px 0 8px 0;
     border-radius: 3px;
     border: none;
 }
 
+.customQuizButton {
+    text-align: center
+}
+
+#editQuizSelect {
+    width: 78%;
+}
+
+#editQuizButton {
```

Comment:
Most of the elements you are styling here are not part of your `QuizBuilder` class but are referenced in your `index.html`. Therefore you should also put these styles in the `styles.css`.

The rule to know which file a style should go into is pretty simple: If the CSS selector you are using is also used inside the code of the `QuizBuilder` class, then it needs to go into `quizBuilderCSS.css`, if not then not.

Reply:
done in https://github.com/sik-leanix/100-days-of-javascript/pull/2/commits/fd45d0aae0bdc1c3506e1dd11317be4889c8aa30